src/services/contracts/IDContract.ts

```diff
@@ -328,6 +328,12 @@ class IDContract {
             version: idData.version,
         };
     }
+
+    async getChainID() {
```

Comment:
this is more of a system level function, nothing to do with IDContract. so better to put in `eosio.ts`

Also, there is a lot of valuable information in this API endpoint. It would be better to provide a service which is used like this `getChainInfo().chain_id` IMO